adapter/OutputBuffer.h

```diff
@@ -0,0 +1,38 @@
+/**********************************************************************************************
+ *                                                                                            *
+ *       CalculiX adapter for heat transfer coupling and mechanical FSI using preCICE         *
+ *       Heat transfer adapter developed by Lucía Cheung with the support of SimScale GmbH    *
+ *                                                                                            *
+ *       Adapter extended to fluid-structure interaction by Alexander Rusch                   *
+ *                                                                                            *
+ *********************************************************************************************/
+
+#ifndef CCX_OUTPUTBUFFER_H
+#define CCX_OUTPUTBUFFER_H
+
+// To avoid including CalculiX.h
+#ifdef LONGLONG
+#define ITG long long
```

Comment:
I guess it's "integer" since it's either int or long, so it's a macro to accept 64-bits indexes.
Not sure it's documented anywhere, it's mostly a guess.